misc/peer-id-generator/src/main.rs

```diff
@@ -67,13 +66,11 @@ fn main() {
     for _ in 0..num_cpus::get() {
         let prefix = prefix.clone();
         thread::spawn(move || loop {
-            let private_key: [u8; 32] = rand::random();
-            let generated = SecioKeyPair::secp256k1_raw_key(private_key).unwrap();
-            let peer_id: PeerId = generated.to_public_key().into_peer_id();
+            let generated = identity::Keypair::new_secp256k1();
+            let peer_id = generated.public().into_peer_id();
             let base58 = peer_id.to_base58();
             if base58[2..].starts_with(&prefix) {
                 println!("Found {:?}", peer_id);
-                println!("=> Private key = {:?}", private_key);
```

Comment:
The while point of this binary is that it prints the private keys that it generates.